### DISCUSSION TITLE
Harman Packaged Browser site lock

### DISCUSSION THREAD

**Discussion by veso266** [airsdk/Adobe-Runtime-Support#650] [Q&A]
Hi there, I was wondering how does Harman Packaged Browser site lock work

is there a mms.cfg or something similar so I can define my own domain in it, because for now Harman Packaged Browser is showing the flash content blocked image because I don't have the right domain

so where can I whitelist my domain then?
Thanks for Anwsering and Best Regards

**Comment by marchbold** [comment]
Please ask your questions in the discussion forum, the issue tracker is for us to report bugs in the AIR SDK.

**Comment by ajwfrost** [comment]
Hi @veso266  - can you find the "hcsfp.dll" or "hcsfp.ocx" file from the package that you have, zip that up and email it to me, and I can take a look? I think emails under 10MB will get through, otherwise please use a file-share thing..   -> andrew.frost at harman.com
thanks
